docs/dxp/7.x/en/site-building/site-appearance/developer-guide/themes/introduction-to-themes.md

```diff
@@ -0,0 +1,50 @@
+# Introduction to Themes
+
+## What is a Theme?
+
+* What is a theme? Is it needed? What can you do with it?
+* Maybe link to the Getting Started article for changing a Site's theme to ground discussion
+
+Maybe reference the "theme anatomy", and if possible show a picture (otherwise link to an article in the same vein)
+
+## Possible Section on Just Using Themes in general in Liferay UI?
+
+[Screenshot of page with one of the non-classic themes?]
+
+Not sure if there's enough content past, how to change a theme (with link), for a whole section... but if so, put that here
+
+## Introduction to Clay and CSS Development?
+
+Not sure if it's needed to go into full detail here, but at least a general overview of how they tie into theme development, and why that's important. Then link to respective articles for more information
+
+## Developing Themes
+
+* How do you get started creating/developing themes?
+    Not getting too deep into explanation, but pointing to an overview of theme development workflow possibly
+
+* Just pointing to some of the major topics and linking out to their respective articles
+
+### Upgrading Themes
```

Comment:
Definitely, upgrading themes is a separated topic from "Developing themes", I think it deserves a place on its own